CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required( VERSION 2.8 )
 
 project( elastix )
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
OK, but I hope it works well with `cmake_minimum_required( VERSION 2.8 )`. You see, `CMAKE_CXX_STANDARD` was introduced with CMake 3.1. But I haven't yet succeeded to upgrade `cmake_minimum_required` (issue #21).